react-hot-button/my-app/src/App.tsx

```diff
@@ -0,0 +1,43 @@
+// import { useState } from 'react'
+import './ButtonContainer.css';
+import './App.css';
+import { ButtonContainer } from './ButtonContainer';
+import { ClickCount } from './ClickCount';
+import { useState } from 'react';
+
+const buttonText = 'Hot Button';
+let className = 'cold';
+function App() {
+  const [count, setCount] = useState(0);
+
+  function handleClick() {
+    setCount(count + 1);
+    if (count === 3) {
```

Comment:
Remember that `count` here is the value it had _before_ you called setCount. So there's a chance that your colors are switching one click too late. (It's too hard for me to figure this out in my head!)